src/logic/tictactoe.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+module.exports = class TicTacToe {
+  constructor () {
+    this.emptyToken = '-'
+    this.tieToken = 'T'
+    this.playerOne = 'X'
+    this.playerTwo = 'O'
+
+    // Start with a clean board
+    this.newGame()
+  }
+
+  newGame () {
+    // Reset the board
+    this.board = {
+      rows: [],
+      moves: 0,
+      maxMoves: 9,
+      currentPlayer: 'X'
+    }
+
+    // Fill the board with emptyTokens
+    for (let i = 0; i < 3; i++) {
+      let row = []
+      for (let j = 0; j < 3; j++) {
+        row.push(this.emptyToken)
+      }
+      this.board.rows.push(row)
+    }
+  }
+
+  makeMove (c) {
```

Comment:
c for char? Perhaps a more descriptive name would be clearer?

Reply:
c is for cell, and yes clearly it can be better named.